Major Constitution Revision (#16)
ritsec/constitution: constitution/article-06.tex

\begin{enumerate}
  \item This position is elected by the club members.
  \item This position is the face of club and E-Board.
  \item Preside over and exercise general supervision over RITSEC events. 
  \item Preside over the E-Board.
  \item Attend or send an alternate member to all auxiliary meetings involving the club.
  \item Maintain the RITSEC E-Board calendar.
\end{enumerate}

% SECTION C: VICE PRESIDENT
\Section{Vice President}

\begin{enumerate}
  \item This position is elected by the club members.
  \item Retained by the President as a supporting role.
  \item Assist in RITSEC endeavors as decided by E-Board.
  \item Preside over the A-Board.

>   \item In charge of all newly created A-Board positions unless, otherwise assigned.

Reviewer: ```suggestion
  \item In charge of all newly created A-Board positions unless otherwise assigned.
```

\end{enumerate}

% SECTION D: SECRETARY
\Section{Secretary}

\begin{enumerate}
  \item This position is elected by the club members.
  \item Responsible for communication between E-Board and club members:
  \begin{enumerate}
    \item Recording and publishing E-Board's weekly meeting notes publicly to the club body.
    \item Managing the livestream and recordings of club meetings and all other qualifying meetings. 
    \item Maintaining RITSEC's public calendar.
  \end{enumerate}
  \item Responsible for communicating with Interest Groups and Programs and their respective chiefs.
  \item Responsible for handling transition of power for Interest Groups.
\end{enumerate}

% SECTION E: TREASURER
\Section{Treasurer}

\begin{enumerate}
  \item This position is elected by the club members.

>   \item Responsible for all financial transactions of all aspects of the organization and keep all financial records up to date. This includes, but is not limited to:

Reviewer: ```suggestion
  \item Responsible for all financial transactions of all aspects of the organization and keeping all financial records up to date. This includes, but is not limited to:
```

  \item Assumes the role of Operations Chief.
\end{enumerate}

% SECTION K: HEAD OF EDUCATION
\Section{Head of Education}

\begin{enumerate}
  \item This position is elected by the club members.

>   \item Supervise club activities within the education portion of a club meeting.

Reviewer: ```suggestion
  \item Supervise club activity during the education portion of club meetings.
```

  \item Ensure a presentation has been created for the security topic of the week.
  \item Assist the Tech Lead with weekly demos.
\end{enumerate}

% SECTION L: HEAD OF RESEARCH
\Section{Head of Research}

\begin{enumerate}
  \item This position is elected by the club members.

>   \item Supervise club activities within the research portion of a club meeting.

Reviewer: ```suggestion
  \item Supervise club activity during the research portion of club meetings.
```
